include/aos/common/cryptoutils.hpp

```diff
@@ -21,12 +21,13 @@ namespace cryptoutils {
 class CertLoader {
 public:
     /**
-     * Constructs object instance.
+     * Initializes object instance.
```

Comment:
Should be merged with cryptoutils implementation commit.

Reply:
That's a way too complicated now, because of dependent commit, eg UUIDManager removal.